### PR TITLE
Feature/use es6

### PR DESCRIPTION
**Pull request by darly-r**
1) Create constant with Array of book objects.
2) Use sort array methods.
3) Use Template literals.
4) Use one modal window for all books

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,198 +10,41 @@
...
-    </div> <!-- End of modal -->
-  
-        <div class="row bg-secondary text-white">
-          <div class="col-lg-1">100004</div>
-          <div class="col-lg-3">THE THIRTEENTH TALE</div>
-          <div class="col-lg-2">Diane Setterfield</div>
-          <div class="col-lg-1">£29.89</div>
-          <div class="col-lg-2"><button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#detailsModal04">View details</button></div> 
-      </div>
+  <div id="listAll"></div>
```
Comment (the-daria) [inline index.html]: Please use kebab-style here

File: index.html [hunk 2]
```diff
@@ -10,198 +10,41 @@
...
-        <div class="row">
-          <div class="col-lg-1">100005</div>
-          <div class="col-lg-3">The Girl on the Train</div>
-          <div class="col-lg-2">Paula Hawkins</div>
-          <div class="col-lg-1">£19.79</div>
-          <div class="col-lg-2"><button type="button" class="btn btn-outline-light text-dark" data-toggle="modal" data-target="#detailsModal05">View details</button></div>
+<!-- <div class="col-lg-2"><button type="button" class="btn btn-outline-light text-dark" data-toggle="modal" data-target="#detailsModal05">View details</button></div> -->
 <!-- Modal -->
-    <div class="modal fade" id="detailsModal05" role="dialog">
+    <div class="modal fade" id="detailsModal" role="dialog">
```
Comment (the-daria) [inline index.html]: modal-details sounds better ;-)

File: index.html [hunk 3]
```diff
@@ -10,198 +10,41 @@
...
       <div class="modal-dialog">
 
 <!-- Modal content -->
         <div class="modal-content">
           <div class="modal-header">
-            <h4 class="modal-title">The Girl on the Train</h4>
+            <h4 id="modalTitle" class="modal-title">empty</h4>
```
Comment (the-daria) [inline index.html]: kebab-style

File: index.html [hunk 4]
```diff
@@ -10,198 +10,41 @@
...
             <button type="button" class="close" data-dismiss="modal">&times;</button>
           </div>
           <div class="modal-body"> 
 
             <div class="container-fluid">
             <div class="row">
-              <div class="col-3"><img src="images/penumbra.jpg" alt="error load" width="100" height="150">
+              <div class="col-3"><img id="imgSource" src="images/penumbra.jpg" alt="error load" width="100" height="150">
```
Comment (the-daria) [inline index.html]: book-img

File: src/buildList.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
+'use strict';
+const arrayOfBooks = 
+[
+	{id: 10001,
+	name: "Mr. Penumbras 24-Hour Bookstore",
```
Comment (the-daria) [inline src/buildList.js]: Use single quotes when you use simple string, it's not a rule but it is common usage on the most of projects https://stackoverflow.com/questions/242813/when-to-use-double-or-single-quotes-in-javascript

File: src/buildList.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    document.getElementById("listAll").innerHTML ="";
+    viewList();
+};
+function sortByPriceRev() {
+    arrayOfBooks.reverse(function(a, b){return a.price - b.price});
+    document.getElementById("listAll").innerHTML ="";
+    viewList();
+};
+function sortByID() {
+    arrayOfBooks.sort(function(a, b){return a.id - b.id});
```
Comment (the-daria) [inline src/buildList.js]: use arrow function here and in other places

File: src/buildList.js [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+};
+
+function sortByPrice() {
+    arrayOfBooks.sort(function(a, b){return a.price - b.price});
+    document.getElementById("listAll").innerHTML ="";
+    viewList();
+};
+function sortByPriceRev() {
+    arrayOfBooks.reverse(function(a, b){return a.price - b.price});
+    document.getElementById("listAll").innerHTML ="";
```
Comment (the-daria) [inline src/buildList.js]: Single quotes

File: src/buildList.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    {id: 10005,
+	name: 'The Girl on the Train',
+	author: 'Paula Hawkins',
+	price: 19.79,
+	imgSrc: "images/trainGirl.jpg",
+	modalBody: `<p>Hardcover, 552 pages. Published March 14th 2006 by Alfred </p>`
+	},
+
+];
+
```
Comment (the-daria) [inline src/buildList.js]: Remove unnecessary empty strings

File: src/buildList.js [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+
+
+
+function viewList(){
+	for (let i = 0; i < arrayOfBooks.length; i++) {
+		let classRow="row bg-secondary text-white";
+		if (i % 2 == 0) {classRow="row bg-light text-dark"} else {classRow="row bg-secondary text-white"};
```
Comment (the-daria) [inline src/buildList.js]: Try ternary operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

**Comment by darly-r** [comment]
Deleted empty strings.
Used:
1) single quotes instead of doble;
2) ternary operator
3) arrow functions
4) kebab-style

**Comment by darly-r** [comment]
exchange double quotes  to single

**Review by the-daria — APPROVED** [review]

**Pull request merged**